Scones/glob.py

```diff
@@ -0,0 +1,34 @@
+def glob(env, root: str, extensions: list):
+    """
+    Finds all files in a root directory matching the provided file extensions
+
+    Returns a flattened list of File nodes
+    """
+    sources = []
+    for (dirpath, dirs, files) in os.walk(root):
+        for extension in extensions:
+            sources.append(env.Glob(dirpath + '/*' + extension))
```

Comment:
This should have slightly better performance
```suggestion
            sources.append(env.Glob(f'{dirpath}/*{extension}'))
```

Reply:
Addressed in a9bfd261b36f4f272ff1412e79cecbe6b2258d22